jrnl/plugins/datecount_exporter.py

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env python
+# encoding: utf-8
+# Copyright (C) 2012-2021 jrnl contributors
+# License: https://www.gnu.org/licenses/gpl-3.0.html
+
+from .text_exporter import TextExporter
+from .util import get_date_counts
+
+
+class DatecountExporter(TextExporter):
+    """This Exporter can lists the tags for entries and journals, exported as a plain text file."""
+
+    names = ["datecount"]
```

Comment:
Is this the best name for this? The plugin that counts tags is just `tags` so maybe just `dates`? Or `count-by-date` as an more descriptive alternate?

Reply:
I used the name suggested in #743, but `dates` would make more sense. What do you think?

Reply:
Yup, that's my fault for not updating an old issue. I think I like `dates` better.

Reply:
Fair enough. I'll update the name shortly.

Reply:
Fixed.